.changeset/yellow-turkeys-lay.md

```diff
@@ -0,0 +1,10 @@
+---
+"@firebase/firestore-types": minor
+"@firebase/firestore": minor
+"@firebase/webchannel-wrapper": minor
```

Comment:
Can you add `"firebase": minor` here to minor bump the main package?

Reply:
Done